CMakeLists.txt

```diff
@@ -448,10 +448,12 @@ endif()
 
 if(SLANG_ENABLE_PREBUILT_BINARIES)
     if(CMAKE_SYSTEM_NAME MATCHES "Windows")
+		file(GLOB prebuilt_binaries "${CMAKE_SOURCE_DIR}/external/slang-binaries/bin/windows-x64/*")
         add_custom_target(
             copy-prebuilt-binaries ALL
-            COMMAND ${CMAKE_COMMAND} -E copy_directory
-                ${CMAKE_SOURCE_DIR}/external/slang-binaries/bin/windows-x64
+			COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_BINARY_DIR}/$<CONFIG>/${runtime_subdir}
+            COMMAND ${CMAKE_COMMAND} -E copy_if_different 
+                ${prebuilt_binaries}
                 ${CMAKE_BINARY_DIR}/$<CONFIG>/${runtime_subdir}
```

Comment:
I didn't realize they added this!

The [CMake documentation](https://cmake.org/cmake/help/latest/manual/cmake.1.html) says that this was added in 3.26. The slang CMakeLists.txt has a minimum requirement of 3.20. 

Is using this worth the minimum requirement bump?

Reply:
Let's not bump it just yet. The problem with new cmake is that it requires the user to build from source instead of just pulling from the default apt-get repo, that adds friction.

For windows this is less an issue...